sources/tiff/setup.py

```diff
@@ -44,7 +44,7 @@ def prerelease_local_scheme(version):
     ],
     install_requires=[
         'large-image',
-        'libtiff>=0.4.1',
+        'pylibtiff>=0.4.1',
```

Comment:
We should probably remove the version limiter.  `pylibtiff` was only published as 0.4.4 (and our wheel which uses an artificially higher number to get priority).

Reply:
Resolved in 42167f341a4286b0d0bcce0ae8aa4ab0ab844d84.